Docker_Container/README.md

```diff
@@ -0,0 +1,13 @@
+## Just put the whole project inside this folder
+
+1. First install Remote Development from extension
+
+2. You will see that an icon is displayed on the bottom-left corner of the VS Code window.
+
```

Comment:
Is this procedure written for the CLI or a GUI. If it is a GUI, which one is it? Please specify.